README.md

```diff
@@ -127,7 +127,9 @@ flow-merge --help
 You can display the config yaml schema and the default values by running:
 
 ```bash
-flow-merge schema | jq
+flow-merge schema
```

Comment:
Removed the jq as default since it might easily throw error if not installed, but kept them as 'extra tip' as they would be useful if checking the schema out.